performance_test/src/experiment_configuration/experiment_configuration.cpp

```diff
@@ -319,6 +319,14 @@ void ExperimentConfiguration::setup(int argc, char ** argv)
     if (vm.count("db_port")) {
       m_db_port = vm["db_port"].as<unsigned int>();
     }
+    if (!vm.count("db_user") || !vm.count("db_password") || !vm.count("db_host") ||
+      !vm.count("db_port"))
+    {
+      m_use_odb = false;
```

Comment:
Instead of this, you could create a new argument with [`program_options::bool_switch`](https://www.boost.org/doc/libs/1_37_0/doc/html/boost/program_options/bool_switch.html)? That way the user can specify if they want to use it, rather than inferring they don't want to use it because those arguments weren't set.

Reply:
The way it's done now restores the old behavior when the arguments aren't set.